src/integrationTest/java/uk/gov/hmcts/reform/pip/subscription/management/config/SwaggerPublisherTest.java

```diff
@@ -24,9 +24,8 @@
 @SpringBootTest(classes = {Application.class},
```

Comment:
Think it might work by just using @SpringBootTest without the following bits in the bracket. This test does not need to spin up the web server?

Reply:
Updated :)